bundles/sirix-rest-api/src/main/kotlin/org/sirix/rest/SirixVerticle.kt

```diff
@@ -237,7 +240,7 @@ class SirixVerticle : CoroutineVerticle() {
         put("/:database").consumes("application/xml").coroutineHandler {
             Auth(keycloak, authz, AuthRole.CREATE).handle(it)
             it.next()
-        }.handler(BodyHandler.create()).coroutineHandler {
```

Comment:
You've added the _BodyHandler_ to all POST requests. However, this one is a PUT request. I wonder if we miss something!? Not sure, though, as the tests pass, and the tests, as far as I can remember, issue a PUT when a resource is created.